examples/soupbintcp-client/README.md

```diff
@@ -15,20 +15,20 @@ an Unsequenced Data packet and receiving an Sequenced Data packet.
 
 ## Results
 
-The following results are from openSUSE 15.0 running Darwin 17.7 and OpenJDK 8:
+The following results are from openSUSE 15.0 running GNU/Linux 4.12.14 x86_64 and OpenJDK 8:
 
 ```
 Warming up...
 Benchmarking...
 Results (n = 50000):
 
-      Min:      35.30 µs
-   50.00%:      48.29 µs
-   90.00%:      77.44 µs
-   99.00%:      79.62 µs
-   99.90%:     107.52 µs
-   99.99%:     233.09 µs
-  100.00%:    1909.76 µs
+      Min:      11.18 µs
+   50.00%:      12.65 µs
+   90.00%:      13.49 µs
+   99.00%:      24.45 µs
+   99.90%:      30.67 µs
+   99.99%:      38.50 µs
+  100.00%:      75.84 µs
```

Comment:
Sorry, just a short question: was there a specific change done that lead to this massive improvement or is this just new hardware? ;) It is not stated which hardware was used (or I missed it).

Thanks,
Chris.

Reply:
I think the hardware (and the operating system?) might have changed.

For this reason on `master` the performance test [has been run on AWS c4.xlarge](https://github.com/paritytrading/nassau/tree/master/tests/soupbintcp-perf-test) in order to make it easy to reproduce the test results.

Reply:
As @jvirtanen noted, it is purely hardware -- the first one was an (old) MacBook Air, the second on my Linux development machine.

Reply:
The development metrics have now been reverted to the original.